kse/src/main/resources/org/kse/gui/dialogs/resources_es.properties

```diff
@@ -569,6 +569,9 @@ DViewPrivateKey.jbAsn1.tooltip       = Mostrar volcado ASN.1 para la clave priva
 DViewPrivateKey.jbExport.mnemonic    = X
 DViewPrivateKey.jbExport.text        = Exportar
 DViewPrivateKey.jbExport.tooltip     = Exportar la clave privada a un archivo
+DViewPrivateKey.jbGenCert.mnemonic   = G
+DViewPrivateKey.jbGenCert.text       = Generar
+DViewPrivateKey.jbGenCert.tooltip    = Generar certificado autofirmado
```

Comment:
Hi @jgrateron 

FYI it seems they are no longer used.

See [d46f69b](https://github.com/kaikramer/keystore-explorer/pull/610/commits/d46f69b55b14d7d7caaec2431f83d02a3cbffe40)

And you modification on:
- #602

Regards,
Th.